client/src/views/SignupNonProfit.tsx

```diff
@@ -1,7 +1,836 @@
-import { SignUpUserAndNonprofit } from '../components/Users/Auth/SignUpUserAndNonprofit/SignUpUserAndNonprofit';
+/* eslint-disable @typescript-eslint/no-unused-vars */
+/* eslint-disable no-unused-vars */
+// import { SignUpUserAndNonprofit } from '../components/Users/Auth/SignUpUserAndNonprofit/SignUpUserAndNonprofit';
+// function SignupNonProfit() {
+//   return <SignUpUserAndNonprofit />;
+// }
+// export default SignupNonProfit;
+
+import * as React from 'react';
+import Grid from '@mui/material/Grid';
+import Input from '@mui/material/Input';
+import Button from '@mui/material/Button';
+import Checkbox from '@mui/material/Checkbox';
+import Typography from '@mui/material/Typography';
+import FormControl from '@mui/material/FormControl';
+import FormControlLabel from '@mui/material/FormControlLabel';
+import type { Theme } from '@mui/material/styles';
+import { useContext } from 'react';
```

Comment:
same idea re: consolidating the imports ... we can probably replace the top line with 
```
import React, { useContext, useEffect } from 'react';
```